[static-runtime] a pass through checks throwing exceptions

tenpercent opened the pull request:
Summary: increasing verbosity where possible

Test Plan: CI

Differential Revision: D43761268

davidberard98 reviewed (approved):
LGTM! (but make sure to take a look at the issues raised by the linters)

tenpercent committed:
[static-runtime] a pass through checks throwing exceptions (#95983)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/95983

increasing verbosity where possible

Test Plan: CI

Reviewed By: davidberard98

Differential Revision: D43761268

fbshipit-source-id: 78db6b61116191d32222099c22cb91af756f76c6
